include/metal/list/sort.hpp

```diff
@@ -3,13 +3,15 @@
 
 #include "../config.hpp"
 #include "../detail/sfinae.hpp"
+#include "../lambda/lambda.hpp"
 #include "../list/list.hpp"
 #include "../number/if.hpp"
+#include "../number/less.hpp"
 
 namespace metal {
     /// \cond
     namespace detail {
-        template<typename lbd>
+        template<typename lbd = metal::lambda<metal::less>>
```

Comment:
Looks great, do you think you can add a couple test cases to make sure it can be instantiated with a single argument as long as it's a list of numbers?

I know all taht preprocessor [voodoo](https://github.com/brunocodutra/metal/blob/master/test/unit/include/test/preprocessor.hpp#L133) is inscrutable and could possibly, but I'm happy to help with any questions you may have.

If you're interested, you may want to have a look at [this summary of some of the techniques it uses](https://github.com/pfultz2/Cloak/wiki/C-Preprocessor-tricks,-tips,-and-idioms) before you dive in.

Reply:
Also, could you add a sentence about this feature to the docs?